tests/PeclEvLoopTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @author Ivan Kalita kaduev13@gmail.com
+ */
+
```

Comment:
@jsor, thanks, I'll keep that in mind next time :smile:. I removed them in 54de798.